src/08_comprehensions.py

```diff
@@ -11,23 +11,21 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+y = [i for i in range(1,6)]
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
-
+y = [pow(i, 2) for i in range(0,10)]
```

Comment:
They are looking for cubes here, I also use ** in place of pow() `y = [x**3 for x in range(10)]`